pkg/policy/validate.go

```diff
@@ -530,6 +613,15 @@ func validateResources(rule kyverno.Rule) (string, error) {
 	return "", nil
 }
 
+// makes no sense to add this since we would have to have the same err != nil checks above too
+// func validateResourcesMatchHelper(rmr kyverno.ResourceFilter) (string, error) {
+// 	// matched resources
+// 	if path, err := validateMatchedResourceDescription(rmr.ResourceDescription); err != nil {
+// 		return fmt.Sprintf("match.resources.%s", path), err
+// 	}
+// 	return "", nil
+// }
+
```

Comment:
Please remove unused code.

Reply:
> As discussed, let's remove this check and log a separate issue.


If we remove this then the policies with "all" won't get applied since this function needs to return `false` in order for the policy to get applied.